src/game_api/entity.hpp

```diff
@@ -129,6 +129,7 @@ struct EntityDB
     int32_t sound_killed_by_other;
     float field_a8;
     int32_t field_AC;
+    /// Array of [Animation](#Animation) types, had to use `pairs` loop to loop thru

```

Comment:
`had to use `pairs` loop to loop thru` seems like redundant information, you always have to use `pairs` or `ipairs` to loop and this being a map makes it clear that `ipairs` will not suffice

Reply:
that's not a type, it's typedef

Reply:
I don't care whether that's a type or a typedef, the auto-docs is just copying strings. So I would assume it says `AnimationMap` so the docs are clear in communicating that it is a map, hence iteration with `pairs` is implied. And I just checked the docs, they do indeed say that. That was my only point, the docs you added are redundant, but they don't hurt so if you have strong feelings about it just keep them.